presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -206,6 +207,37 @@ public void testArrayContains()
     }
 
     @Test
+    public void testArrayJoin()
+            throws Exception
+    {
+//        assertFunction("ARRAY_JOIN(ARRAY [1, 2, 3], ';', 'N/A')", VARCHAR, "1;2;3");
```

Comment:
uncomment these. Also, update them to return the null argument at the end, so that it tests the version with only two args